tensorflow/c/kernels_test.cc

```diff
@@ -291,6 +293,133 @@ TEST_F(TestKernelAttr, StringList) {
   CreateAndCallKernelWithAttr(my_create_func, "TestKernelAttrStringList", v);
 }
 
+TEST_F(TestKernelAttr, Tensor) {
+  struct TensorProtoData : public ::tensorflow::TensorProto {
```

Comment:
This seems to break internal builds since `::tensorflow::TensorProto` is marked final (not sure why the OSS tests pass). Could we avoid subclassing `TensorProto` here?

Reply:
Done!